src/pages/iou/request/step/IOURequestStepDescription.js

```diff
@@ -61,6 +67,12 @@ function IOURequestStepDescription({
         }, []),
     );
 
+    let currentDescription = lodashGet(transaction, 'comment.comment', '');
+
+    if (iouType === CONST.IOU.TYPE.SPLIT && action === CONST.IOU.ACTION.EDIT && !lodashIsEmpty(splitDraftTransaction)) {
+        currentDescription = lodashGet(splitDraftTransaction, 'comment.comment', '') 
+    } 
```

Comment:
```suggestion
    // In the split flow, when editing we use SPLIT_TRANSACTION_DRAFT to save draft value
    const isEditingSplitBill = iouType === CONST.IOU.TYPE.SPLIT && action === CONST.IOU.ACTION.EDIT;
    const currentDescription =
        isEditingSplitBill && !lodashIsEmpty(splitDraftTransaction) ? lodashGet(splitDraftTransaction, 'comment.comment', '') : lodashGet(transaction, 'comment.comment', '');
```

1. Move this to be before the the `useFocusEffect`
2. In the updateComment condition use `isEditingSplitBill`